### ISSUE TITLE
Android: toggling keyboard in the post-session feedback modal shows flickering content in the background.

### ISSUE THREAD

**Issue by Greenheart** [29ki/29k#2683]
### What brings you here and how are you?

Android: toggling keyboard in the post-session feedback modal shows flickering content in the background.

https://github.com/29ki/29k/assets/6125097/8a5308e8-293e-416d-bff3-07a92c2a1822

**Comment by gewfy** [comment]
@Greenheart This one doesn't affect the functionality of it right?

**Comment by Greenheart** [comment]
Exactly, the modal works as expected. This is a minor fix to make the app feel more polished :)
